src/main/java/games/strategy/triplea/settings/ClientSetting.java

```diff
@@ -147,9 +147,6 @@ public static void flush() {
     }
   }
 
-  /**
-   * Returns true if a value has been set for the current property.
-   */
```

Comment:
I noticed a superfluous Javadoc on this overridden method while I was here, so I went ahead and removed it.  The superclass method is properly Javadoced.